### PR TITLE
Example: Safe + 4337 + Passkeys example application

### PR DESCRIPTION
**Pull request by mmv08**
This PR adds an example application where a Safe with a passkey owner is deployed and minting an NFT. Implements https://github.com/safe-global/safe-modules/issues/188 and https://github.com/safe-global/safe-modules/issues/187

Demo recording:
https://www.loom.com/share/c44946ad137341daa9be08dce07cd886

Testing the application:
1. Follow the instructions in the README

### CODE REVIEW

**Comment by nlordell** [comment]
> No contracts are verified 😞. Sorry!

I chuckled.

File: examples/safe-4337-passkeys/.env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Get projectId at https://cloud.walletconnect.com
+VITE_WC_CLOUD_PROJECT_ID=
+// 4337 Bundler URL. We recommend https://www.pimlico.io/
+VITE_WC_4337_BUNDLER_URL=
```
Comment (nlordell) [inline examples/safe-4337-passkeys/.env.example:4]: nit: missing newline at EOF.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ This repository contains a collection of modules for the [Safe Smart Account](ht
 - [4337 Module](./modules/4337)
 - [Allowance Module](./modules/allowances)
 
+## Examples
+
+- [Safe + 4337 + Passkeys](./examples/safe-4337-passkeys)
```
Comment (nlordell) [inline README.md:12]: Unrelated to this PR, but I feel like the gas metering project should also be moved here. Maybe there is a more general directory name to “examples” that would fit both workspace packages...

File: examples/safe-4337-passkeys/src/App.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  /* text-align: center; */
```
Comment (nlordell) [inline examples/safe-4337-passkeys/src/App.css:5]: Left over commented code?

File: .eslintrc.js [hunk 4]
```diff
@@ -1,18 +1,21 @@
 module.exports = {
+  root: true,
```
Comment (nlordell) [inline .eslintrc.js:2]: Would it be possible to “inherit” from these ESLint settings for the `safe-4337-passkeys` project specifically? This way we don’t need to pull in additional ESLint plugins for the packages in `modules/*` that don’t need them?

**Review by nlordell — COMMENTED** [review]
LGTM, just a few small nits (**disclaimer**: I didn’t look too closely at the FE code). It just occurred to me, but should this live in `5afe` org?

File: .eslintrc.js (hunk 4 above)
Comment (mmv08) [reply .eslintrc.js:2]: Have added a config in the passkeys app file that extends this one.

**Review by nlordell — APPROVED** [review]

**Pull request merged**